Libraries/LibWeb/HTML/Scripting/Environments.h

```diff
@@ -124,6 +124,9 @@ struct EnvironmentSettingsObject : public Environment {
     // https://w3c.github.io/ServiceWorker/#get-the-service-worker-object
     GC::Ref<ServiceWorker::ServiceWorker> get_service_worker_object(ServiceWorker::ServiceWorkerRecord*);
 
+    // https://w3c.github.io/webappsec-credential-management/#active-credential-types
+    Vector<String> active_credential_types() const;
```

Comment:
my first instinct is to have this return `Vector<String> const&` so that callers are not forced to copy the types if they don't need to.